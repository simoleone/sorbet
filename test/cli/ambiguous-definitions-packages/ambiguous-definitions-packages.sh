#!/bin/bash

set -e

cd test/cli/ambiguous-definitions-packages || exit 1

../../../main/sorbet --silence-dev-message --stripe-mode --stripe-packages . 2>&1


