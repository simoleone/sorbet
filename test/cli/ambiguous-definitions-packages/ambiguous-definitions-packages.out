foo/a.rb:5: Class definition is ambiguous https://srb.help/5068
     5 |    module Foo::D # ambiguous
     6 |    end
    foo/a.rb:4: Alternate definition here.
     4 |  module Foo
          ^^^^^^^^^^
Errors: 1
